opal/mca/base/mca_base_var_enum.c

```diff
@@ -460,7 +460,6 @@ int mca_base_var_enum_create_flag(const char *name, const mca_base_var_enum_valu
         return OPAL_ERR_OUT_OF_RESOURCE;
     }
 
-#if OPAL_ENABLE_DEBUG
     int all_flags = 0;
```

Comment:
Thank you for finding/fixing this.  Can you make this line be:

```suggestion
    int __opal_attribute_unused__ all_flags = 0;
```

This will prevent a compiler warning about `all_flags` being unused in a production build.